src/w3d/renderer/mapper.h

```diff
@@ -123,17 +129,438 @@ class LinearOffsetTextureMapperClass : public ScaleTextureMapperClass
 
     void Set_UV_Offset_Delta(const Vector2 &per_second)
     {
-        m_uvOffsetDeltaPerMS = per_second;
-        m_uvOffsetDeltaPerMS *= -0.001f;
+        m_UVOffsetDeltaPerMS = per_second;
+        m_UVOffsetDeltaPerMS *= -0.001f;
     }
 
-    void Set_Sync_Time(uint32_t time) { m_lastUsedSyncTime = time; }
-    uint32_t Get_Sync_Time() { return m_lastUsedSyncTime; }
+    void Set_Sync_Time(unsigned int time) { m_lastUsedSyncTime = time; }
+    unsigned int Get_Sync_Time() { return m_lastUsedSyncTime; }
 
 protected:
     Vector2 m_currentUVOffset;
-    Vector2 m_uvOffsetDeltaPerMS;
```

Comment:
unneeded changes
we make the first word lower case in our naming scheme, if the first word is short for something then that becomes lower case
like UV -> uv, D3D -> d3d